src/User_Command.php

```diff
@@ -418,6 +418,54 @@ public function create( $args, $assoc_args ) {
 	 * <user>...
 	 * : The user login, user email or user ID of the user(s) to update.
 	 *
+	 * --user_pass=<login>
```

Comment:
Can you make each of these arguments optional? `[--user_pass=<login>]` instead of `--user_pass=<login>`